alpha/lib/model/MediaServerNode.php

```diff
@@ -137,4 +137,9 @@ public function getExplicitLiveUrl($liveUrl, LiveStreamEntry $entry)
 		return $this->getUrlType() . "/$userType/";
 	}
 
+    public function getAdditionalUrlParam(LiveStreamEntry $entry) : string
```

Comment:
having return declaration type wont work on low php versions

Reply:
Move return type to comment